python/simulation.py

```diff
@@ -162,6 +162,15 @@ def __init__(self, center, size=Vector3(), direction=mp.AUTOMATIC, weight=1.0):
         self.weight = complex(weight)
 
 
+class ModeRegion(object):
```

Comment:
This seems like a separate class, not an alias. What’s wrong with just ModeRegion=FluxRegion?

Reply:
Fixed.